utils/mockData.ts

```diff
@@ -0,0 +1,17 @@
+export const products = [
```

Comment:
I recommend to define a type for products

```suggestion
export const products: Array<Product> = [
```